workitem/simple_type.go

```diff
@@ -94,6 +95,15 @@ func (fieldType SimpleType) ConvertToModel(value interface{}) (interface{}, erro
 		default:
 			return nil, errors.Errorf("value %v should be %s, but is %s", value, "MarkupContent", valueType)
 		}
+	case KindCodebase:
+		if valueType.Kind() != reflect.Map {
+			return nil, errors.Errorf("value %v should be %s, but is %s", value, reflect.Map, valueType.Name())
+		}
+		validCodebase, err := codebase.NewCodebase(value.(map[string]interface{}))
+		if err != nil {
+			return nil, err
+		}
+		return validCodebase.ToMap(), nil
```

Comment:
@aslakknutsen yes, I tried that but unable to convert `map[string]interface{}` to `CodebaseContent`. Casting panics.